community/modules/compute/schedmd-slurm-gcp-v6-nodeset/versions.tf

```diff
@@ -20,7 +20,7 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 3.83"
+      version = ">= 5.11"
```

Comment:
How did you pick 5.11. Would it work with 4.48. I believe that it would avoid the failing test. It looks like the field we care about is there: https://registry.terraform.io/providers/hashicorp/google/4.84.0/docs/resources/compute_reservation

Reply:
The one you are referring is resource, we want to check the data for the particular reservation. Which is supported from 5.11.